launcher/launcher/main.go

```diff
@@ -100,20 +94,6 @@ func main() {
 	if err := os.MkdirAll(launcherfile.HostTmpPath, 0744); err != nil {
```

Comment:
You need to move this line above the GetLaunchSpec

Reply:
good catch